python_src/src/lamp_py/performance_manager/l0_rt_trip_updates.py

```diff
@@ -7,23 +7,22 @@
 from lamp_py.runtime_utils.process_logger import ProcessLogger
 
 from .gtfs_utils import (
-    start_time_to_seconds,
-    add_static_version_key_column,
+    add_missing_service_dates,
     add_parent_station_column,
-    unique_trip_stop_columns,
+    add_static_version_key_column,
     rail_routes_from_filepath,
+    start_time_to_seconds,
+    unique_trip_stop_columns,
 )
 
 
 def get_tu_dataframe_chunks(
-    to_load: Union[str, List[str]], db_manager: DatabaseManager
+    to_load: Union[str, List[str]], route_ids: List[str]
```

Comment:
changed this to make testing easier.

now i can test this function without needing to standup a whole ass database.